data/lang/module-combat/en.json

```diff
@@ -189,7 +189,7 @@
   },
   "HARVESTING_LAND_THERE": {
     "description": "",
-    "message": "We have a salesperson in every nearby star system. This person will get in contact with you when landed."
+    "message": "We have a salesperson in every nearby star system. This person will get in contact with you when landed. Please note that we will not pay you if we receive the result of the mission after the due date!"
```

Comment:
Sounds like it's addressing the issue a bit too much.
hand-wavey suggestion:
"...when landed for payment before due date".